app/scripts/modules/core/instance/instance.write.service.js

```diff
@@ -6,8 +6,72 @@ module.exports = angular
   .module('spinnaker.core.instance.write.service', [
     require('../task/taskExecutor.js'),
     require('../serverGroup/serverGroup.read.service.js'),
+    require('../utils/lodash.js'),
   ])
-  .factory('instanceWriter', function (taskExecutor, serverGroupReader) {
+  .factory('instanceWriter', function (taskExecutor, serverGroupReader, _) {
+
+    function buildMultiInstanceJob(instanceGroups, type) {
+      let jobsByZone = instanceGroups
+        .filter((instanceGroup) => instanceGroup.instances.length > 0)
+        .map((instanceGroup) => {
+          /*
+          // TODO: ttomsu - when zone and launchTimes aren't required, skip the groupBy stuff below and just use this
+           return {
+             type: type,
+             cloudProvider: instanceGroup.cloudProvider,
+             providerType: instanceGroup.cloudProvider,
+             instanceIds: instances.map((instance) => instance.id),
+             credentials: instanceGroup.account,
+             region: instanceGroup.region,
+             serverGroupName: instanceGroup.serverGroup
+           };
+           */
```

Comment:
Not sure I grok what's going on here - you're constructing a single call from Deck that will be fanned out in Orca? And that call will contain multiple jobs, grouped by...zone? region?

If this is operating on instances, why is the grouping necessary at all?